include/srtp.h

```diff
@@ -1424,7 +1408,7 @@ typedef enum {
 
 typedef struct srtp_event_data_t {
   srtp_t        session;  /**< The session in which the event happend. */
-  srtp_stream_t stream;   /**< The stream in which the event happend.  */
+  uintptr_t     ssrc;     /**< The ssrc in host order of the stream in which the event happend */
```

Comment:
@paulej , I'm not sure was just trying to be carefully. Someone could possibly be printing the value ... Could change to use a uint32_t now and be done with it.